File: java/src/main/java/ai/buble/sdk/chat/ChatModelList.java
package ai.buble.sdk.chat;

import java.util.List;

/**
 * OpenAI-style chat model list.
 */
public class ChatModelList {
    private String object;
    private List<ChatModel> data;

    public String getObject() { return object; }
    public void setObject(String object) { this.object = object; }
    public List<ChatModel> getData() { return data; }
    public void setData(List<ChatModel> data) { this.data = data; }
}

File: dotnet/src/Buble.Sdk/Chat/ChatModelList.cs
using System.Text.Json;
using System.Text.Json.Serialization;

namespace Buble.Sdk.Chat;

public sealed class ChatModelList
{
    [JsonPropertyName("object")]
    public string? Object { get; set; }

    [JsonPropertyName("data")]
    public IReadOnlyList<ChatModel>? Data { get; set; }

    [JsonExtensionData]
    public Dictionary<string, JsonElement>? AdditionalProperties { get; set; }
}
